### DISCUSSION TITLE
Welcome to AquaPi Discussions!

### DISCUSSION THREAD

**Discussion by TheRealFalseReality** [TheRealFalseReality/aquapi#17] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by DaveCBr** [comment]
Nick- can you post info on the pumps you use for waste and top off?

**Reply by DaveCBr** [reply]
Figured it out after loading your blueprint.  Ordering two (actually three) of the pumps.  

One question however; how do you prevent the water drain pump from syphoning water when not running? Is your waste water holding tank higher than your display tank?

**Comment by timlovely63-code** [comment]
I had that same problem. David, I ended up just pulling my hose above the
water level of my reservoir.  I know places like Fall Creek so backflow
valves for that kind of tubing.

**Comment by TheRealFalseReality** [comment]
@DaveCBr @timlovely63-code The pumps I use/sell are Peristaltic Pumps, they use negative pressure. When not running, they hold the liquid in the tubes, and they do not back siphon.

**Reply by DaveCBr** [reply]
Thanks Nick.  I did order two today from Am.z..n.  The kits were on discount.  I am setting up for water changes and ATO for now.  Dosing maybe later.

**Comment by DaveCBr** [comment]
Nick - Just as quick question - What happened to the AquaPi thread on the R2R forum?  is it blocked or did it get removed?  

<img width="627" height="156" alt="image" src="https://github.com/user-attachments/assets/03ca52d0-b486-4854-b3e4-41a354f35c77" />

**Reply by TheRealFalseReality** [reply]
Hmmm, I'm looking into it, thanks!

**Comment by TheRealFalseReality** [comment]
they wont let me post it there as they believe its a commercial thread and
they want me to support them.

*Nicholas White*
*Capital City Aquatics - Tallahassee's Aquarium Services*
We do the wet work so you don’t have to!

850-347-5981
Our Website <http://capitalcityaquatics.com>
Facebook <https://www.facebook.com/OfficialCapitalCityAquatics/>  - YouTube
<https://www.youtube.com/channel/UCevO8FlyEdGPLRvNoAixH5A> - TikTok
***@***.***?lang=en> - Instagram
<http://instagram.com/capitalcityaquatics>

**Comment by DaveCBr** [comment]
Ah. Makes sense I guess.  Maybe use a different platform. Or just use GIT
or spin up a Discord channel.

Also take a look at my post in your AI app area in GIT.  The AI portion of
that driver has errors in HA and will not report the narrative.  The
balance is running like reporting temps and water level.

Also at some point, I want to learn how you are doing water change.  Same
pump as your make up water pump?  I assume not since it’s salt vs fresh
water


Best Regards,
David Bruner

**Comment by TheRealFalseReality** [comment]
Hmmm, try. Do you think a Discord server would help? It would a another
direct way to talk to me. Github is the best btw.

*Nicholas White*
*Capital City Aquatics - Tallahassee's Aquarium Services*
We do the wet work so you don’t have to!

850-347-5981
Our Website <http://capitalcityaquatics.com>
Facebook <https://www.facebook.com/OfficialCapitalCityAquatics/>  - YouTube
<https://www.youtube.com/channel/UCevO8FlyEdGPLRvNoAixH5A> - TikTok
***@***.***?lang=en> - Instagram
<http://instagram.com/capitalcityaquatics>

**Comment by TheRealFalseReality** [comment]
I dunno How I missed the issues in the AI Home Assistant!

*Nicholas White*
*Capital City Aquatics - Tallahassee's Aquarium Services*
We do the wet work so you don’t have to!

850-347-5981
Our Website <http://capitalcityaquatics.com>
Facebook <https://www.facebook.com/OfficialCapitalCityAquatics/>  - YouTube
<https://www.youtube.com/channel/UCevO8FlyEdGPLRvNoAixH5A> - TikTok
***@***.***?lang=en> - Instagram
<http://instagram.com/capitalcityaquatics>

**Comment by TheRealFalseReality** [comment]
For large water changes, nothing replacing the siphon and all. But I have
setup 2 dosing pumps that remove and add water from the tank at 5 gallons a
day. The pumps need some work so right now they are out of commission. The
first prototype (which I am using) has an issue with the power cable
clamping that I fixed but just not one my own pumps yet lol.

*Nicholas White*
*Capital City Aquatics - Tallahassee's Aquarium Services*
We do the wet work so you don’t have to!

850-347-5981
Our Website <http://capitalcityaquatics.com>
Facebook <https://www.facebook.com/OfficialCapitalCityAquatics/>  - YouTube
<https://www.youtube.com/channel/UCevO8FlyEdGPLRvNoAixH5A> - TikTok
***@***.***?lang=en> - Instagram
<http://instagram.com/capitalcityaquatics>

**Comment by DaveCBr** [comment]
I think in Discord you would not get into an issue lime with R2R with the
product support and so on.  GIT works great and that would work as well.



Best Regards,
David Bruner

**Comment by DaveCBr** [comment]
Thanks.  Let me know if you need me to test anything.  I don’t really know
when it stopped (or what version of HA) caused the issue.  Note that I am
on the beta of HA.



Best Regards,
David Bruner

**Reply by TheRealFalseReality** [reply]
I prefer Git! But I wonder if Discord is easier if more users are on there. I just don't have discord open enough to follow through with that yet. 

What issue are you referring to? Did something stop working?

**Comment by DaveCBr** [comment]
Nick
You fixed the AI issue with your last update.  All is good with both
systems.

I think Discord is worth a shot.  It’s far less intimidating than GIT for
users that are not coders.



Best Regards,
David Bruner

**Reply by TheRealFalseReality** [reply]
Ah that issue, yes!
